### PR TITLE
Added worker for using incense

### PR DESCRIPTION
**Pull request by bmorrise**

### CODE REVIEW

**Comment by brantje** [comment]
Looks good!

File: pokemongo_bot/cell_workers/use_incense.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+              'use_incense',
+              formatted="Using {type} incense. {incense_count} incense remaining",
+              data={
+                  'type': self.types.get(type, 'Unknown'),
+                  'incense_count': self.bot.item_inventory_count(type)
+              }
+          )
+        else:
+          self.emit_event(
+              'use_incense',
```
Comment (TheSavior) [inline pokemongo_bot/cell_workers/use_incense.py]: I think we typically use a different event for the failed state. @douglascamata is that right?

File: pokemongo_bot/cell_workers/use_incense.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+from pokemongo_bot.item_list import Item
+
+class UseIncense(BaseTask):
+    SUPPORTED_TASK_API_VERSION = 1
+
+    def initialize(self):
+      self.start_time = 0
+      self.use_incense = self.config.get('use_incense', False)
+      self.use_order = self.config.get('use_order', {})
+      self.incense_ordinary_count = self.bot.item_inventory_count(Item.ITEM_INCENSE_ORDINARY.value)
```
Comment (TheSavior) [inline pokemongo_bot/cell_workers/use_incense.py]: The way this is written right now, you will only be calculating this on startup. `initialize` is only run on bot startup`. Everything else should be done in `work`, or functions called by `work`.

File: pokemongo_bot/__init__.py [hunk 3]
```diff
@@ -1002,10 +1003,9 @@ def heartbeat(self):
             pass
 
     def update_web_location_worker(self):
-        pass
-        # while True:
-        #     self.web_update_queue.get()
-        #     self.update_web_location()
+        while True:
```
Comment (TheSavior) [inline pokemongo_bot/__init__.py]: This was changed in another PR.

**Comment by TheSavior** [comment]
@bmorrise Why close?

File: pokemongo_bot/cell_workers/use_incense.py (hunk 1 above)
Comment (douglascamata) [inline pokemongo_bot/cell_workers/use_incense.py]: @TheSavior yes, you're right.

**Pull request closed without merging**